<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDNI">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>97</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDni">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>75</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI/RUC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDireccion">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>126</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dirección:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTelefono">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>190</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Teléfono:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDireccion">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>154</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTelefono">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>209</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCorreo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFecha">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEstado">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstado">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFecha">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha de Registro:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCorreo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Correo:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardarCliente">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCodigo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblCodigo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Codigo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
